xml/release-notes.xml

```diff
@@ -513,31 +513,23 @@
 
  </sect1>
 
- <!-- <sect1 xml:id="sec.driver">
+ <sect1 xml:id="sec.driver">
   <title>Drivers and Hardware</title>
   <para/> -->
 
-  <!-- <sect2 xml:id="sec.driver.nvidia-hybrid">
-   <!-/- boo#1094834 -/->
-   <title>Hang on Machines with Nvidia GPUs and Hybrid Graphics</title>
+  <sect2 xml:id="sec.driver.sig">
+   <!-/- bsc#173158 -/->
+   <title>CONFIG_MODULE_SIG=y</title>
    <para>
-    With the kernel shipped in &thisflavor; 15.0 GM, the Nouveau driver for
-    Nvidia graphics card may hang at reboot, shutdown, or during runtime power
-    management actions. This bug occurs primarily on system with hybrid
-    graphics, such as laptops that include integrated Intel graphics and a
-    discrete Nvidia graphics card.
+     This is an important change for all users of 3rd party drivers using secureboot.
+     The module signature check is on &thisflavor; 15.2.
+     Unsigned modules will be tainted and will result in a warning message.
+     Signed modules that fail to check will be blocked.
```

Comment:
Sorry, but I fail to understand what this actually means. If e.g. NVidia drivers for 15.2 are signed against 15.2 how can their check fail? And why would there be unsigned modules? 
And further, what was the previous behavior and how and why has it changed?

Reply:
Well the entire point is to use drivers signed by vendor or unsigned. If you try a some driver which seems like it's signed but it was altered (somebody tries to inject something). Then it's right thing to block it.